### DISCUSSION TITLE
Simple HA button to turn off the soundbar

### DISCUSSION THREAD

**Discussion by Soiski71** [PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar#72] [Q&A] [answered]
Hi, I thought it would be easy to turn the soundbar easily off by generation a button (card) to do this. But I have not found a way. Only showing it as possibility in the standard card that it created .
![image](https://github.com/user-attachments/assets/4fe09926-5f65-4f0b-a62b-520d8612c0f2).

I have tried on the scripts but if trying to do any actions on the soundbar it does not allow it ?

![image](https://github.com/user-attachments/assets/c6c7be4f-6063-485a-b03e-dd81ae78c5aa)

Any suggestions ?

**Comment by PiotrMachowski** [comment, marked as answer]
You can just use a service `media_player.turn_off`

**Reply by Soiski71** [reply]
Thanks it was too obvious (still learning HA)
